src/components/DpMultistepNav/DpMultistepNav.vue

```diff
@@ -3,6 +3,7 @@
     <button
       v-for="(step, idx) in steps"
       :key="`step_${idx}`"
+      :data-cy="`${dataCy}:${idx}`"
```

Comment:
```suggestion
      :data-cy="`${dataCy}:${step.label}`"
```

Reply:
With this change we have now hooks like: `data-cy="multistepNav:Ihre Stellungnahme"` with space.

Can I make it like `:data-cy="`${dataCy}:${step.label.replace(/\s+/g, '-').toLowerCase()}`"`?
Then Hook `data-cy="multistepNav:ihre-stellungnahme"`

Reply:
https://github.com/demos-europe/demosplan-ui/pull/881/commits/44354c98fe426a948a840edba3aa62a2ca3a6fd2

Reply:
Ok. Sorry. Thought the label would look more like a key. than we better stick to the idx.